src/components/Subscription/PublicCalendarSubscriptionPicker.vue

```diff
@@ -21,22 +21,32 @@
 
 <template>
 	<NcModal @close="$emit('close', $event)">
-		<div class="holiday-subscription-picker">
-			<h2>{{ t('calendar', 'Public holiday calendars') }}</h2>
-			<p class="holiday-subscription-picker__attribution">
-				{{ t('calendar', 'Public holiday calendars are provided by Thunderbird. Calendar data will be downloaded from {website}', { website: 'thunderbird.net' }) }}
+		<div class="public-calendar-subscription-picker">
+			<h2 v-if="showHolidays">
+				{{ t('calendar', 'Public holiday calendars') }}
+			</h2>
+			<h2 v-else>
+				{{ t('calendar', 'Public calendars') }}
+			</h2>
+			<p v-if="showHolidays" class="holiday-subscription-picker__attribution">
+				{{ t('calendar',
+					'Public holiday calendars are provided by Thunderbird. Calendar data will be downloaded from {website}',
+					{ website: 'thunderbird.net' }) }}
 			</p>
-			<div v-for="calendar in calendars" :key="calendar.source" class="holiday-subscription-picker__region">
-				<div class="holiday-subscription-picker__region__name">
-					<h3>{{ calendar.country }}</h3>
-					<div class="holiday-subscription-picker__region__name__subline">
-						{{ calendar.datespan }}
+			<p v-else class="holiday-subscription-picker__attribution">
+				{{ t('calendar', 'These public calendars are suggested by the sever administrator. Calendar data will be downloaded from the respective website.') }}
+			</p>
+			<div v-for="calendar in calendars" :key="calendar.source" class="public-calendar-subscription-picker__region">
```

Comment:
Can you handle `calendars` being empty? Using [`NcEmptyContent`](https://nextcloud-vue-components.netlify.app/#/Components/NcEmptyContent).